src/inverseRobot.js

```diff
@@ -4,10 +4,31 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInversed = {};
+  let robotValues = [];
+
+  robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let j = 0; j < robotValues.length; j++) {
+      if (i === j) {
+        continue;
+      }
+
+      if (robotValues[i] === robotValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    robotInversed[robot[key]] = key;
+  }
```

Comment:
please create variable for `robot[key]`
```suggestion
  for (const key in robot) {
    if(check if property already exists) { if exists do something }
    other than that - do something else
    robotInversed[robot[key]] = key;
  }
```